freqtrade/templates/FreqaiExampleStrategy.py

```diff
@@ -195,8 +195,8 @@ def populate_indicators(self, dataframe: DataFrame, metadata: dict) -> DataFrame
         # each training period.
         dataframe = self.model.bridge.start(dataframe, metadata, self)
 
-        dataframe["target_roi"] = dataframe["&-s_close_mean"] + dataframe["&-s_close_std"] * 1.25
-        dataframe["sell_roi"] = dataframe["&-s_close_mean"] - dataframe["&-s_close_std"] * 1.25
+        dataframe["target_roi"] = dataframe["&-s_close_mean"] + dataframe["&-s_close_std"] * 1.1
```

Comment:
What is the purpose of this change?

Reply:
Hey, can you please remove the documentation changes from this branch/PR? The other PR will be merged separately.